packages/EasyActivity/src/Interfaces/ActivitySubjectInterface.php

```diff
@@ -11,12 +11,17 @@ public function getActivitySubjectId(): string;
     public function getActivitySubjectType(): string;
 
     /**
-     * @return array<string>|array<string, mixed>
+     * @return list<string>|array<string, list<string>>
      */
-    public function getAllowedActivityProperties(): ?array;
+    public function getAllowedActivityProperties(): array;
 
     /**
-     * @return array<string>|array<string, mixed>
+     * @return array<string, list<string>>
      */
-    public function getDisallowedActivityProperties(): ?array;
+    public function getNestedObjectAllowedProperties(): array;
```

Comment:
Let's make this method name consistent with the others: `getNestedObjectAllowedActivityProperties`.

Reply:
done